README.md

```diff
@@ -11,7 +11,7 @@ The recommended method of installation is [through composer](http://getcomposer.
 ```JSON
 {
     "require": {
-        "pschwisow/phergie-irc-plugin-react-url-shorten": "dev-master"
+        "pschwisow/phergie-irc-plugin-react-url-shorten": "^1"
```

Comment:
People actually lock in their version of Composer? Hrm. Odd, considering running `composer self-update` is all it takes, with the option of running `composer rollback` to revert to the previous version.

In any case, `~1` would be an acceptable substitute.